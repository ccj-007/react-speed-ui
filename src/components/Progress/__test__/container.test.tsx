import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Progress } from '../../index';

describe('test Progress component', () => {
  it('should render', () => {
    const { asFragment } = render(<Progress percent={0} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
