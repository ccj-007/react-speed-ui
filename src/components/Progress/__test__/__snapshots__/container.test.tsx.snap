// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Progress component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-progress-bar"
  >
    <div
      class="speed-progress-bar-outer"
      style="height: 15px;"
    >
      <div
        class="speed-progress-bar-inner color-primary speed-progress-bar-inner-right css-16krv3a-Progress"
        style="width: 0%;"
      >
        <span
          class="inner-text"
        >
          0%
        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
